api/db/migrations/20230612092952_add-version-column-on-certification-course.js

```diff
@@ -0,0 +1,23 @@
+const TABLE_NAME = 'certification-courses';
+const COLUMN_NAME = 'version';
+
+const up = async function (knex) {
+  await knex.schema.table(TABLE_NAME, function (table) {
+    table.integer(COLUMN_NAME);
```

Comment:
Pinaillage: pour être moins gourmand éventuellement

```suggestion
    table.tinyint(COLUMN_NAME);
```